server/admin_management/tests/routers/test_router_access_control_privilege.py

```diff
@@ -171,6 +173,60 @@ def test_get_access_control_privileges_by_application_id(
     )
 
 
+def test_assign_new_bceid_user_delegated_admin_save_business_guid(
```

Comment:
Hi Ian, this is small, but just thinking of our naming consistent for our tests, maybe we could rename it to "test_create_access_control_privilege_for_bceid_user_should_save_business_guid" ?  Just feel normally we name our test case based on "test_[method name]_[conditions]"?

Reply:
Yes, I know we normally did that, but I was struggling that will have long name on test method so instead just give a name that's meaningful.
It's sometime even harder to name the condition in a short name. But for sure, I change it if you feel we should stick to that.